### PR TITLE
Fixes #2352, Japanese translation by a Native Japanese

### LINKED ISSUE #2352: Translate App Strings to Japanese
### User Story

```Please note: we have no speakers of this language on the team, so it's difficult to check the validity of translations. Therefore, on issues like this, we will wait for at least 2 PRs before merging anything (within a sane timeframe). If you submit a PR, please also justify why your translation is better than any that have previously been submitted, so we can make an informed choice about which to merge. In the case that we need to use elements of both, we will most likely reward bounties to both contributors, for making and verifying the translations.```

As a Japanese language speaker, I want to be able to interact with Status in my native language.

### Description

*Type*: Feature

*Summary*: We need someone fluent (and eloquent) in Japanese to translate all the latest strings in Status from English. We would like to invite our community members to take on these tasks in order
a) To become a bit more acquainted with the Status repos and perhaps wander further into the codebase and set of issues we need help with, earning yourself Open Bounties all the way down this rabbit hole, and
b) To make sure that the product we deliver has been built by the same community that uses it, in a virtuous feedback loop of increasing understanding and value-generation.

#### Expected behavior
The Japanese translation ought to be as up-to-date as the English one.

#### Actual behavior
We have not yet updated it, preferring to leave this open as an opportunity for our community to get involved again.

### Solution

*Summary*: 

- The file [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/ja.cljs) needs to be updated to match the new English one [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/en.cljs).
- How you translate is a matter of your preference. There were significant updates to the English file for the 0.9.12 release, with lots of strings both add and REMOVED. We need you to make sure that the files are exact mirrors of each other, except in different languages.
- It might be easier to simply take the new English file and translate from scratch rather than going through the current translated files line by line (though we leave this up to you).

### Additional Information

* Status version: 0.9.12+

There are bounties available for these translations. Please see the QR codes that will be added below after the DevCon 3 announcement.

### PR DESCRIPTION
**Pull request by CrypoNinja**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

If you submit PR for issue with bounty then write here Fixes #NN where NN is issue number

*otherwise*

addresses #...

### Summary:

[comment]: # (Summarise the problem and how the pull request solves it)
...

### Steps to test:
- Open Status
- ...
- Step 3, etc.

[comment]: # (PRs will only be accepted if squashed into single commit.)

status: ready

*or*

status: wip

### CODE REVIEW

**Comment by andytudhope** [comment]
Hi! This has been picked up, but we will need you to find someone else to vouch for and verify your translation as per the issue description, as there is no-one on our team who speaks Japanese.

**Review by kazoocoin — APPROVED** [review]
looks good

**Review by kazoocoin — APPROVED** [review]
looks great.

**Comment by CrypoNinja** [comment]
got an approval- what else needs to be done?

**Comment by andytudhope** [comment]
I'm afraid neither of you have really verifiable GH profiles. I will chat to the team and see what they say. For our Italian translation, for instance, this was the guy who did the review: https://github.com/vigliag

It's easier for us to trust PRs from people with more active profiles. Moreover, I cannot believe that a review found no changes that needed to be made - no-one gets the translation right first time round and, in proper reviews, there is always some discussion about which phrasing and tone most accurately reflects the intention.

File: src/status_im/translations/ja.cljs
Comment (andytudhope) [inline src/status_im/translations/ja.cljs:6]: Why did you add these spaces? Shouldn't be an issue, but please delete so all content lines up for better readability

**Comment by oskarth** [comment]
@CrypoNinja @andytudhope What's the state of this PR?

**Comment by andytudhope** [comment]
Still waiting for a response with further changes from @CrypoNinja - seems mostly like a Google translate effort with little review, so we cannot merge for now.

**Comment by CrypoNinja** [comment]
HEY ANDY,

STOP insulting me with your stupid guess. I am native japanese and spent more than 3 hours. no google translate, ZERO. Finding another reviewer is NOT my job. Understand????

**Comment by CrypoNinja** [comment]
I did a few other project like this bounty and got very good feedback. Not using my translation is showing your stupidity. and again you are insulting me with no reason.

**Comment by andytudhope** [comment]
I apologise for offending you.

However, I still stand by my original comment - "I cannot believe that a review found no changes that needed to be made - no-one gets the translation right first time round and, in proper reviews, there is always some discussion about which phrasing and tone most accurately reflects the intention." This suggests, to me, that a proper review has not been done, which is a requirement for these translation issues.

Also, shouting at the team will not get you anywhere.

**Comment by oskarth** [comment]
Closing this PR due to inactivity. Feel free to re-open with a motivation if you feel like this was in error.

**Pull request closed without merging**
